dev-lang/nasm/nasm-2.15.05.ebuild

```diff
@@ -31,12 +31,26 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-2.15-bsd-cp-doc.patch
 )
 
+DOCS=(
+	AUTHORS
+	README.md
+	LICENSE
+	INSTALL
+	ChangeLog
+	CHANGES
+	nasm.txt
+	ndisasm.txt
+	version
+	SubmittingPatches
+)
+
 src_compile() {
 	default
 	use doc && emake doc
 }
 
 src_install() {
 	default
+	use doc && einstalldocs
```

Comment:
`einstalldocs` is already called by `default`.

Reply:
> einstalldocs is already called by default.

Shouldn't we make that process conditional? Since end-users seldom read software related documentation.

Reply:
If a user really doesn't want this documentation to be installed, they can use `FEATURES="nodoc"`.

Reply:
I wasn't known of that feature. I'll check out :)

Reply:
You can find a list if you run `man 5 make.conf` and search for `FEATURES` :). There are a few interesting ones.